src/passes/Outlining.cpp

```diff
@@ -13,23 +13,225 @@
 #define DBG(statement)
 #endif
 
+// Check a Result or MaybeResult for error and call Fatal() if the error exists.
+#define ASSERT_ERR(val)                                                        \
```

Comment:
Actually, now that I think about it again, `ASSERT_OK` would be a much better name for this.

Reply:
done